src/sections/Members-grid/Maintainers.js

```diff
@@ -18,6 +18,7 @@ const Maintainer = props =>{
                             twitter
                             status
                             meshmate
+                            Linkedin
```

Comment:
Would you mind if the "L" was lowercased to stay inline with the others, @vinayaksh42?